com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/FilesApiServiceImpl.java

```diff
@@ -102,13 +101,10 @@ public Response uploadMtaFile(HttpServletRequest request, SecurityContext securi
                 continue; // ignore simple (non-file) form fields
             }
 
-            InputStream in = null;
-            try {
-                in = item.openStream();
+            try (InputStream in = item.openStream()) {
                 FileEntry entry = fileService.addFile(spaceGuid, item.getName(), getConfiguration().getFileUploadProcessor(), in);
                 uploadedFiles.add(entry);
-            } finally {
-                IOUtils.closeQuietly(in);
+            } catch (IOException e) { // Handle IO exceptions quietly
```

Comment:
This is now different, though. The previous code's intent was to handle IOExceptions during the closing of the `InputStream`. Now, it also handles/ignores IOExceptions when opening the stream as well. This may result in problems where we report that something was uploaded, but it actually wasn't.

Reply:
Should the error propagate?